### DISCUSSION TITLE
Report Icon

### DISCUSSION THREAD

**Discussion by cake03** [elrumo/macOS_Big_Sur_icons_replacements#1318] [General]
![user-images.githubusercontent.com/80169337/139051430-6a1e83c0-896f-4f6f-bc56-e690bb21314a.png](https://user-images.githubusercontent.com/80169337/139051430-6a1e83c0-896f-4f6f-bc56-e690bb21314a.png)

**Comment by elrumo** [comment]
I am going to create a section on the website to report icons, which is long overdue, I will also need to specify the criteria needed for reporting. In this particular case, what would you say the reason for reporting it would be?

**Reply by SocietasEvanescentes** [reply]
He never replied, but I would say for it being either a meme, or low effort.
